css-containment/Overview.bs

```diff
@@ -208,7 +212,8 @@ Paint Containment</h3>
 		the UA can directly skip trying to paint its contents,
 		as they're guaranteed to be off-screen/obscured as well.
 
-	2. If the containing element is ''overflow:visible'',
+	2. If the containing element would have been ''overflow:visible'',
+		it becomes ''overflow:clip'' and
```

Comment:
This line sounds like normative instructions; I'd prefer a parenthetical like (but is now overflow:clip due to paint containment), or something more direct, like "If the containing element's overflow has been changed to 'clip' (because it was originally visible), ..."